vector/src/main/res/values/strings.xml

```diff
@@ -2566,8 +2572,20 @@
     <string name="auth_pin_reset_title">Reset pin</string>
     <string name="auth_pin_new_pin_action">New pin</string>
     <string name="auth_pin_reset_content">To reset your PIN, you\'ll need to re-login and create a new one.</string>
+    <string name="settings_security_application_protection_title">Protect access to the application.</string>
+    <string name="settings_security_application_protection_summary">Protect access to the application using pin code and biometrics.</string>
+    <string name="settings_security_application_protection_screen_title">Configure protection</string>
     <string name="settings_security_pin_code_title">Enable PIN</string>
     <string name="settings_security_pin_code_summary">If you want to reset your PIN, tap Forgot PIN to logout and reset.</string>
+    <string name="settings_security_pin_code_use_biometrics_title">Use biometrics</string>
+    <string name="settings_security_pin_code_use_biometrics_summary_on">Depending on your device, you will be able to unlock using fingerprints, face recognition, iris recognition, etc.</string>
```

Comment:
**Suggestion 1**
Enable device specific biometrics. ~Depending on your device, you will be able to unlock using fingerprints, face recognition, iris recognition, etc.~
_Don't think its helpful to list the ways biometrics might work_

**Suggestion 2**
Enable device specific biometrics, like fingerprints and face recognition ~Depending on your device, you will be able to unlock using fingerprints, face recognition, iris recognition, etc.~
_Another way to say some biometrics without making a long list_

Reply:
I take suggestion 2 :)